db/migrate/20181028093455_create_repositories.rb

```diff
@@ -0,0 +1,17 @@
+class CreateRepositories < ActiveRecord::Migration[5.2]
+  def change
+    create_table :repositories do |t|
+      t.belongs_to :organization, index: true
+      t.string :uid
+      t.string :name
+      t.string :github_url
+      t.string :desc
+      t.string :license_name
+      t.string :license_spdx_id
```

Comment:
You need to add index here, since your scope (`without license`) will be very inefficient otherwise